tobias/utils/motifs.py

```diff
@@ -199,7 +231,7 @@ def from_file(self, path):
 		#Check correct format of pfms
 		for motif in self:
 			nuc, length = np.array(motif.counts).shape
-			motif.length = length
+
 			if nuc != 4:
 				sys.exit("ERROR: Motif {0} has an unexpected format and could not be read".format(motif))

```

Comment:
This probably became obsolete since you now check the shape at the creation of OneMotif, right?